challenges/6-converter-segundos/javascript/YgorIsmael/ex6.js

```diff
@@ -0,0 +1,9 @@
+// =D
+const tempo = process.argv[2];
+function conversao(valor){
+    horas = Math.floor(valor/3600).toString().padStart(2, '0');
+    minutos = Math.floor((valor%3600)/60).toString().padStart(2, '0');
+    segundos = Math.floor((valor%3600)%60).toString().padStart(2, '0');
+console.log(`${horas}:${minutos}:${segundos}`);
```

Comment:
Você precisa declarar as variaveis com let ou const  na linha 4, 5, e 6.
Estou utilizando o repl para rodar o codigo e me retorna:  ReferenceError: horas is not defined
Na linha 7, ja que você esta chamando a função com o console.log() na linha 9, simplesmente coloque um return dentro da função.

Reply:
Mudei lá vlw

Reply:
O return não precisa de parenteses, mesmo assim o codigo esta funcionando.